types/office-js-preview/index.d.ts

```diff
@@ -15127,8 +15127,9 @@ declare namespace Office {
          * - In compose mode you must call the `saveAsync` method before you can use the `restUrl` member.
          * Your add-in must have **read/write item** permissions to call the `saveAsync` method.
          * However, in delegate or shared scenarios, you should instead use the `targetRestUrl` property of the
-         * {@link Office.SharedProperties | SharedProperties} object (introduced in requirement set 1.8). For more information,
-         * see the {@link https://learn.microsoft.com/office/dev/add-ins/outlook/delegate-access | shared folders and shared mailbox} article.
+         * {@link https://learn.microsoft.com/javascript/api/outlook/office.sharedproperties#outlook-office-sharedproperties-targetresturl-member | SharedProperties}
```

Comment:
you don't need to change this one as it does exist in the preview reqt. set

Reply:
Good catch. Reverted changed.